samples/ar-model-viewer/src/main/kotlin/io/github/sceneview/sample/armodelviewer/MainFragment.kt

```diff
@@ -59,6 +60,7 @@ class MainFragment : Fragment(R.layout.fragment_main) {
             onTrackingChanged = { _, isTracking, _ ->
                 actionButton.isGone = !isTracking
             }
+            editModes = setOf(Transformable.EditMode.ROTATE, Transformable.EditMode.SCALE)
```

Comment:
Replace with `isEditable: Boolean get()=editableTransforms.size > 0 set(value) editableTransforms =...` and `editableTransforms: Set<Transform>` (Transform.POSITION, Transform.ROTATION,...)